plugin-api/build.gradle

```diff
@@ -50,7 +44,7 @@ processResources {
 shadowJar {
   configurations = [project.configurations.runtimeClasspath]
```

Comment:
what about adding `archiveBaseName = 'sonar-plugin-api'` so that the name matches what is published to maven?

Reply:
maybe also `archiveClassifier = ''` too

Reply:
Good idea!